chunks/tool-yugabytedb/Dockerfile

```
@@ -38,7 +38,7 @@ ENV YCQL_API_PORT=12000
 
 # re-initialization is automatically handled
 RUN echo "\n# yugabytedb start command" >> /home/gitpod/.bashrc.d/100-yugabyedb-launch
-RUN echo "yugabyted start --base_dir=$STORE --listen=$HOST" >> /home/gitpod/.bashrc.d/100-yugabyedb-launch
+RUN echo "[[ -f \${GITPOD_REPO_ROOT}/.nopreload ]] || yugabyted start --base_dir=$STORE --listen=$HOST > /dev/null" >> /home/gitpod/.bashrc.d/100-yugabyedb-launch
```

```suggestion
RUN echo "[[ ${DISABLE_AUTO_YB_START} == "true" ]] || yugabyted start --base_dir=$STORE --listen=$HOST > /dev/null" >> /home/gitpod/.bashrc.d/100-yugabyedb-launch
```
IMO using an environment variable for the check makes more sense compared to a file as it is a usual practice in docker world to enable or disable a service using env variables.

---

My understanding is env variables are terminal-specific in Gitpod unless you create a project (which may not be the case for everyone). When you launch a new terminal inside VSCode, it's going to get reset. To me .nopreload file in the source repo is similar to .gitignore. Including that will nullify the default convention.

---

How many cases would you like to use this? Could you consider supporting only those with high usage and using a custom image for the others?